wolfssl/wolfcrypt/error-crypt.h

```diff
@@ -227,10 +232,10 @@ enum {
     CRYPTOCB_UNAVAILABLE= -271,  /* Crypto callback unavailable */
     PKCS7_SIGNEEDS_CHECK= -272,  /* signature needs verified by caller */
     PSS_SALTLEN_RECOVER_E=-273,  /* PSS slat length not recoverable */
+    CHACHA_POLY_OVERFLOW =-274,  /* ChaCha20Poly1305 limit overflow */
+    ASN_SELF_SIGNED_E   = -275,  /* ASN self-signed certificate error */
```

Comment:
Can we keep -274 as self signed error and made chacha poly overflow 275?

Reply:
@dgarske, It's better to keep the same errno as DO-178.